cli/tests/test_operations.rs

```diff
@@ -2862,7 +2862,627 @@ fn modify_git_repo(git_repo: gix::Repository) -> gix::Repository {
     git_repo
 }
 
+#[test]
```

Comment:
Since `test_operations.rs` is big, I think it's better to add `test_operation_revert_command.rs` (or `test_op_revert_command.rs`.)

Reply:
done :+1: